vtclean.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // see regex.txt for a slightly separated version of this regex
-var vt100re = regexp.MustCompile(`^\033([\[\]]([\d\?]+)?(;[\d\?]+)*)?(.)`)
+var vt100re = regexp.MustCompile(`^\033([\[\]]([\d\?]+)?(;[\d\?]*)*)?([m@GCDPK;l])`)
```

Comment:
I'm honestly not thrilled about `([m@GCDPK;l])`, however leaving it as `(.)` broke this test as the `r` would get matched
```
"ccc \033]4;1;rgb:38/54/71test":       "ccc rgb:38/54/71test",
```